src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -329,6 +328,61 @@ SendCommandToMetadataWorkersParams(const char *command,
 }
 
 
+/*
+ * SendCommandToWorkersOptionalInParallel sends the given command to workers in parallel.
+ * It does error if there is a problem while sending the query, but it doesn't error
+ * if there is a problem while executing the query.
+ */
+void
+SendCommandToWorkersOptionalInParallel(TargetWorkerSet targetWorkerSet, const
```

Comment:
General observation: This file is a bit of a mess in terms of naming. There are various similarly named functions doing wildly different things. For instance, SendCommandToWorker and SendCommandToAllWorkers are transactional, but SendCommandToAllWorkers is not.

Reply:
We should probably do the refactoring in another PR